custom/crediblex/portfolio/loc/src/main/java/com/crediblex/fineract/portfolio/loc/data/LineOfCreditRequest.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.crediblex.fineract.portfolio.loc.data;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Schema(description = "Request DTO for Line of Credit operations")
+public class LineOfCreditRequest {
+
+    @Schema(example = "34", description = "Client ID for the line of credit")
+    private Long clientId;
+
+    @Schema(example = "Business Credit Line", description = "Name of the line of credit")
+    private String name;
+
+    @Schema(example = "Payable", description = "Type of product for the line of credit")
+    private String productType;
+
```

Comment:
These values are way less than what i see captured in the excel and ticket. Please doublecheck and update what is required by Jaya.